### PR TITLE
Tilføjede Fit er Frit (wip)

### PR DESCRIPTION
**Pull request by ScumbagDog**
Lavede lidt latex til "Fit er Frit"-sangen, tilsendt fra en af fitnisserne.

Ikke endelig version, kan godt være der er lidt, der skal pudses af inden den merges ind.

### CODE REVIEW

**Review by JakobTopholt — CHANGES_REQUESTED** [review]
Lgtm, with the exception that I am of the opinion that the last verse could be omitted.

We'll take this up the next F-choir meeting in March.

**Comment by krestenlaust** [comment]
> Lgtm, with the exception that I am of the opinion that the last verse could be omitted.
> 
> We'll take this up the next F-choir meeting in March.

You didn't take it up

**Comment by krestenlaust** [comment]
I agree that the last verse isn't 'tidssvarende', but it's a part of the original song. Maybe we could leave it in, but in italics or maybe in brackets

**Comment by JakobTopholt** [comment]
Kresten is spitting facts, it was not taken up. I'll perhaps look at this later.

File: main.tex [hunk 1]
```diff
@@ -88,6 +88,8 @@
 \newpage
 \input{sange/påcassiopeiatilfoobar.tex}
 \newpage
+\input{sange/fiterfrit.tex}
+=======
```
Comment (VirtualSatai) [inline main.tex:92]: Er dette intentionelt eller noget merge conflict som er forblivet ?

Comment (mads256h) [reply main.tex:92]: No way det er intentionelt

Comment (JakobTopholt) [inline main.tex:92]:
```suggestion
\input{sange/fiterfrit.tex}
```

**Review by JakobTopholt — CHANGES_REQUESTED** [review]
Removing the one line it's lgtm.

Also, I removed the last verse some time ago as i didn't like it, feels a bit dictatorial but feel free to send me some feedback on it in the future.

**Review by JakobTopholt — APPROVED** [review]

**Pull request merged**